x-pack/plugins/beats_management/public/components/table/table_type_configs.tsx

```diff
@@ -83,12 +83,48 @@ export const BeatsTableType: TableType = {
     },
     {
       // TODO: update to use actual metadata field
-      field: 'event_rate',
-      name: 'Event rate',
-      sortable: true,
+      field: 'config_status',
+      name: 'Config Status',
+      render: (value: string, beat: CMPopulatedBeat) => {
+        let color: IconColor = 'success';
+        let statusText = 'OK';
+        let tooltipText = 'Everything looks good!';
+
+        switch (beat.config_status) {
+          case 'REQUIRES_UPDATE':
+            statusText = 'Waiting for Beat checkin';
+            tooltipText = 'Waiting for the Beat to download the latest config';
+          case 'WAITING_FOR_SUCCESS':
+            color = 'warning';
+            statusText = 'Verifying Config';
+            tooltipText = 'Waiting for a status update from the Beat';
+            break;
+          case 'UNKNOWN':
```

Comment:
Why not change this to `OFFLINE`? the difference between frontend name and backend name may be misleading

Reply:
I debated this a lot for this reason. I ultimately went with UNKNOWN because it's more "honest" in that we don't really know, it is assumed. The issue in reality could be Kibana. We are in essence casting the type to offline for users to give them our best guess